src/index.html

```diff
@@ -1,14 +1,53 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
+    <meta
+    charset="UTF-8">
+    <meta
+    name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header>
+      <a href="#apple" class="logo"><img src="images/logo.png" alt="pict"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#apple" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#apple" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item actTest">
+            <a href="#apple" class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item actTest">
            <a 
              href="#apple"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers</a>
          </li>
```

If you have 3 or more attributes then move each attribute to the separate line